examples/braket_features/create_custom_circuit_and_custom_local_simulator.ipynb

```diff
@@ -0,0 +1,1296 @@
+{
```

Comment:
<p>Also needs more detail; need to explain what an <code>ExecutionManager</code> is and why it's needed for cirq-ionq </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>